adafruit_ht16k33/ht16k33.py

```diff
@@ -40,10 +40,12 @@
 
 class HT16K33:
     """The base class for all displays. Contains common methods."""
```

Comment:
Since this is a base class for multiple things, I'm not sure it's applicable to expand this docstring to the same extent as, for example, the DotStar lib: https://github.com/adafruit/Adafruit_CircuitPython_DotStar/blob/af25424ee7dbebea3e5d77390c017018ffa52d36/adafruit_dotstar.py#L51 

If it is applicable, we should consider doing it. If it's not, please disregard.

Reply:
I went ahead and put in the `:param` doc. That seems to make sense, and could take care of @sommersoft 's request for documenting `auto_write` default.